src/noshow/dashboard/connection.py

```diff
@@ -76,6 +76,7 @@ def get_patient_list(_session: Session, date_input: date, top_n: int = 20) -> Li
                 <= date.today() - timedelta(days=round(MUTE_PERIOD * 30.5))
             )
         )
+        .where((ApiPatient.opt_out == None) | (ApiPatient.opt_out == 0))  # noqa: E711
```

Comment:
You can solve the E711 error by using `is None` instead of `== None`

Reply:
ChatGPT to the rescue:

In SQLAlchemy, when you want to filter query results based on whether a column's value is not `NULL`, you should use the `is_not()` method rather than trying to use `!=` or `is not` directly with `None`. This approach is specifically designed to handle `NULL` checks in a way that is compatible with the SQL generated by SQLAlchemy.

Here's how you can do it:

```python
from sqlalchemy import create_engine, MetaData, Table, select

# Assuming you have already defined your engine
engine = create_engine('your_database_connection_string')
metadata = MetaData()

# Assuming you have a table object
table = Table('your_table_name', metadata, autoload_with=engine)

# To select rows where a specific column is not NULL
with engine.connect() as connection:
    query = select([table]).where(table.c.your_column_name.is_not(None))
    result = connection.execute(query)
    
    for row in result:
        print(row)
```

In this example, replace `'your_database_connection_string'`, `'your_table_name'`, and `'your_column_name'` with your actual database connection string, table name, and column name, respectively.

This approach is clean and aligns with SQLAlchemy's abstraction of SQL expressions. It avoids the linting error you encountered with `!= None` and the issue that arises from using `is not None`, which doesn't translate to a valid SQL expression within the context of SQLAlchemy.

Reply:
In classic GPT fashion this wasn't the correct answer, but it send me in the right direction. I needed a replacement for the == comparison instead of !=. Luckey is_(None) also works as a replacement